source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Builders/Command/ChargeLinksCommandBuilder.cs

```diff
@@ -31,12 +32,12 @@ public class ChargeLinksCommandBuilder
             RequestDate = SystemClock.Instance.GetCurrentInstant(),
             Recipient = new MarketParticipantDto
             {
-                MarketParticipantId = Guid.NewGuid().ToString("N"),
+                MarketParticipantId = SeededData.MarketParticipants.GridAccessProviderOfMeteringPoint571313180000000005.Gln,
                 BusinessProcessRole = MarketParticipantRole.GridAccessProvider,
             },
             Sender = new MarketParticipantDto
             {
-                MarketParticipantId = Guid.NewGuid().ToString("N"),
+                MarketParticipantId = SeededData.MarketParticipants.SystemOperator.Gln,
                 BusinessProcessRole = MarketParticipantRole.MeteringPointAdministrator,
```

Comment:
To be realistic, the command `sender` data must be like this:

```suggestion
                MarketParticipantId = SeededData.MarketParticipants.GridAccessProviderOfMeteringPoint571313180000000005.Gln,
                BusinessProcessRole = MarketParticipantRole.GridAccessProvider,
```

Reply:
Fixed